aiosmtpd/handlers.py

```diff
@@ -64,14 +64,13 @@ def _print_message_content(self, peer, data):
                 line = line.decode('utf-8', 'replace')
             print(line, file=self.stream)
 
-    def process_message(self, peer, mailfrom, rcpttos, data, **kws):
+    def process_message(self, session, message, **kws):
```

Comment:
I wonder if we can get rid of `**kws` here and just wrap it all up in the `session` or `message` objects?